src/Sentry/Platforms/Cocoa/Sentry.Cocoa.props

```
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <Using Include="Foundation" />
-    <Using Include="Sentry.CocoaSdk.SentryOptions" Alias="SentryCocoaOptions" />
+    <Using Include="Sentry.CocoaSdk.SentryOptions" Alias="SentryCocoaSdkOptions" />
```

How about?

```suggestion
    <Using Include="Sentry.CocoaSdk.SentryOptions" Alias="SentryCocoaOptions" />
```

Not sure the namespace is `CocoaSdk`, could it be `Cocoa`? Do we call `JavaSdk` or `AndroidSdk` on the other binding?